codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/ConstrainedShapeSymbolProvider.kt

```diff
@@ -103,6 +104,14 @@ class ConstrainedShapeSymbolProvider(
                     base.toSymbol(shape)
                 }
             }
+            is IntegerShape -> {
```

Comment:
Can you add a test for this arm in `ConstrainedShapeSymbolProvider.kt`?

Reply:
Done in https://github.com/awslabs/smithy-rs/pull/2005/commits/35dfb824c2bf8fbf8770a852c833063c68979d1b